include/triton/Conversion/TritonGPUToLLVM/AsmFormat.h

```diff
@@ -0,0 +1,33 @@
+#ifndef TRITON_CONVERSION_TRITON_GPU_TO_LLVM_ASM_FORMAT_H_
+#define TRITON_CONVERSION_TRITON_GPU_TO_LLVM_ASM_FORMAT_H_
+
+#include "mlir/IR/Value.h"
+#include "triton/Dialect/Triton/IR/Dialect.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include <memory>
+#include <string>
+
+namespace mlir {
+class ConversionPatternRewriter;
+class Location;
+
+namespace triton {
+using llvm::StringRef;
+
+static std::string strJoin(llvm::ArrayRef<std::string> strs,
```

Comment:
I would prefer using `inline` instead of `static` here to avoid the duplicate symbol linker error.